Header header

BallonetState basic_state

uint8 errors_cnt		#  
uint8 errors_flag1	#  
uint8 errors_flag2	#  
